workflows-scripts/inventory/item-record-operations/item_recordOperations.jmx

```diff
@@ -0,0 +1,3811 @@
+            <collectionProp name="ModuleController.node_path">
+              <stringProp name="764597751">Test Plan</stringProp>
+              <stringProp name="-424091378">item_recordOperationsView</stringProp>
+              <stringProp name="-1394928069">---------- Flows ----------</stringProp>
+              <stringProp name="1544827186">SC: Delete Item</stringProp>
+            </collectionProp>
+          </ModuleController>
+          <hashTree/>
+        </hashTree>
+      </hashTree>
+      <JSR223Listener guiclass="TestBeanGUI" testclass="JSR223Listener" testname="Log Failed Samples" enabled="false">
+        <stringProp name="cacheKey">true</stringProp>
+        <stringProp name="filename"></stringProp>
+        <stringProp name="parameters"></stringProp>
+        <stringProp name="script">if (!prev.isSuccessful()) {
+	String label = prev.getSampleLabel();
+	String requestUrl = prev.getURL().toExternalForm();
+	String samplerData = prev.getSamplerData();
+	String username = vars.get(&quot;uuid&quot;);
+	String responseCode = prev.getResponseCode();
+	String responseMessage = prev.getResponseMessage();
+	String responseBody = prev.getResponseDataAsString();
+	String responseTime = prev.getTime();
+	String toLog = &quot;&quot;;
+	Date date = new Date(prev.getStartTime());
+	def a = date.format(&quot;yyyy-MM-dd_HH-mm-ss:SSS&quot;);
+
+	toLog = &quot;\nTime: &quot; + a + &quot;\nSampler Name: &quot; + label + &quot;\nRequest URL: &quot; + requestUrl + &quot;\nRequest Data: &quot; + samplerData + &quot;\nUser: &quot; + username + &quot;\nResponse Code &amp; Message: &quot; + responseCode + &quot; &quot; + responseMessage + &quot;\nResponse Time (ms): &quot; + responseTime + &quot;\nResponse body:\n&quot; + responseBody + &quot;\n--------------------------------------------------\n&quot;;
+	File failed_samples = new File(vars.get(&quot;global_BaseDir&quot;) + &quot;/failed_samples_${__machineIP()}.txt&quot;);
+	failed_samples.append(toLog);
+
+	} 
+
+</stringProp>
+        <stringProp name="scriptLanguage">groovy</stringProp>
+      </JSR223Listener>
+      <hashTree/>
+      <TestFragmentController guiclass="TestFragmentControllerGui" testclass="TestFragmentController" testname="---------- Flows ----------" enabled="true"/>
+      <hashTree>
+        <GenericController guiclass="LogicControllerGui" testclass="GenericController" testname="SC: View Item" enabled="true"/>
+        <hashTree>
+          <ModuleController guiclass="ModuleControllerGui" testclass="ModuleController" testname="MC: Click Inventory" enabled="true">
+            <collectionProp name="ModuleController.node_path">
+              <stringProp name="764597751">Test Plan</stringProp>
+              <stringProp name="-424091378">item_recordOperationsView</stringProp>
+              <stringProp name="-1206634253">---------- Raw Test Fragments ----------</stringProp>
+              <stringProp name="408023744">SC: GoTo Inventory (02.xxx)</stringProp>
```

Comment:
Can you please explain what does this naming convention "(02.xxx)" stand for? Is the ".xxx" part being used to distinguish anything?

Reply:
It stands for convenient debugging so it may be removed if necessary

Reply:
I think "02" stands out already, unless you think the ".xxx" part can be replaced with something else?

Reply:
>  ".xxx" part 

it was removed